Kajabi/pine
# chore: upgrade stencil and node

# Description

Upgrade Stencil to the latest version. Based on the requirements of Stencil v4, it was required to use the Node v18.19, since we have been testing 20 and things have been running well. I elected to use this opportunity to move to Node v20.5.0

# How to Test
In the root directory, run the following commands
1. `rm -rf **/node_modules`. This will remove all the node_modules in the subdirectories
2.  If you are using `nvm`, run `nvm install 20.5.0`
3. `nvm use`
4. `yarn install`
5. Start the server, `yarn start`... or however you would if you use Tmux

Things should start as normal. In another terminal window, confirm there are no Git changes.
